pegasus/sites.v3/code.org/views/teach_page_pl.haml

```diff
@@ -0,0 +1,14 @@
+- if request.language == "en"
+  %h2=hoc_s(:teach_page_pl_offerings_english_header)
+-else
+  %h2=hoc_s(:teach_page_pl_offerings_intl_header)
+%div.grid-container
+  - if request.language == "en"
+    = view :teach_page_pl_list_workshops
+  - else
+    = view :teach_page_pl_list_self_paced
+  - if request.language == "en"
+    = view :teach_page_pl_list_self_paced
+  - else
+    = view :teach_page_pl_list_workshops
```

Comment:
Could this be simplified to an if (English) {show workshops; show self-paced} else {show self-paced; show workshops} instead of the two separate conditionals?

Reply:
Ah yes, ty! Just updated.

I knew there had to be a cleaner way to do this 😅